docs/Tutorials/BeginnersTutorial.md

```diff
@@ -146,12 +146,12 @@ directory where you have the `H5` files, run
 
 ```
 spectre generate-xdmf \
-  --file-prefix ExportTimeDependentCoordinates3DVolume \
-  --subfile-name element_data --output BBH_Coords
+  --subfile-name element_data --output BBH_Coords \
+  ExportTimeDependentCoordinates3DVolume*h5
 ```
 
-The `--file-prefix` argument must be whatever is before the number `0` in the
-name of your `Volume` file. We output volume data per node so we append the
+The last argument must be a regular expression covering all h5 files to be
+observed (and no others). We output volume data per node so we append the
```

Comment:
It's any number of positional arguments, they can be glob patterns (not regex), and they don't have to come last in the list of arguments.

Docs like this are hard to keep up-to-date, so maybe just delete the sentence? Your call.